middleware.js

```diff
@@ -0,0 +1,53 @@
+// Exports
+// =============================================================================
+export const config = {
+  matcher: ['/preview/(index.html)?'],
+};
+
+// Rewrite rules shared with local server configurations
+export const rewriteRules = [
+  // Replace CDN URLs with local paths
+  {
+    match: /https?.*\/CHANGELOG.md/g,
+    replace: '/CHANGELOG.md',
+  },
+  {
+    // CDN versioned default
+    // Ex1: //cdn.com/package-name
+    // Ex2: http://cdn.com/package-name@1.0.0
+    // Ex3: https://cdn.com/package-name@latest
+    match: /(?:https?:)*\/\/.*cdn.*docsify[@\d.latest]*(?=["'])/g,
+    replace: '/lib/docsify.min.js',
+  },
```

Comment:
Shall we replace it to `/lib/docsify.js`? I believe it could benefit for debug in dev and preview side.

Reply:
Since all of our templates use `docsify.min.js`, I think that is the file we should test with as well. It's possible that something could go wrong in the minification process, and if we are previewing with the unminified `docsify.js` file we may not catch it.

But your point about debugging is a good one. I think our best option is to generate a source map for `docsify.min.js` so that we can load the minified bundle but review unminified source via the dev tools when an error occurs.

How does that sound?

Reply:
I see, thats fine to me.